README.md

```diff
@@ -1,2 +1,110 @@
-# idm-styleguide
-IDM's living styleguide
+# idm-web-uikit
+
+IDM's living web ui kit.
+
+## Base technology
+
+- React
+- CSS modules
+- [PostCSS](https://github.com/postcss/postcss) with [MOXY's preset](https://github.com/moxystudio/postcss-preset-moxy)
+- SVG spriting support with [external-svg-sprite-loader](https://github.com/karify/external-svg-sprite-loader)
+
+
+## Commands
+
+### start
+
+```sh
+$ npm run storybook
+```
+
+Starts [Storybook](https://storybook.js.org/).
+
+### build
+
+```sh
+$ npm run build
+```
+
+Builds the project.
+
+NOTE: at the current stage of the package, the build script has not yet been defined. It will be defined in the near future.
+
+### lint
+
+```sh
+$ npm run lint
+```
+
+Checks if the project has any linting errors.
+
+### release
+
+```sh
+$ npm run release
+```
+
+Releases and publishes the package. Runs tests, lints and builds the project beforehand.
+
+This command uses [`standard-version`](https://github.com/conventional-changelog/standard-version) underneath. The version is automatically inferred from the [conventional commits](https://conventionalcommits.org/).
+
+NOTE: this package still does not have a release script, as it is still in the initial stages of its development. This script will be added in the near future.
+
+## Using a linked version of idm-web-uikit
+
+In some cases, you may want to make changes to IDM's web-uikit at the same time as you work on your project which uses the web-uikit. In order to use a local version of [idm-web-uikit](https://github.com/ipfs-shipyard/idm-web-uikit) and have any web-uikit modifications be reflected live on your project, some pages have to be made in your main project.
```

Comment:
Perhaps we should add the following: Some of the instructions below assume you are using Webpack in your main project.

Reply:
We should do the same for the discussify style guide as well.

Reply:
Ok. It makes sense. Is the rest ok for you?

Reply:
Yep!